src/ape/utils.py

```diff
@@ -45,14 +45,15 @@ def get_relative_path(target: Path, anchor: Path) -> Path:
     which may or may not share a common ancestor.
     NOTE: Both paths must be absolute
     """
-    assert anchor.is_absolute()
-    assert target.is_absolute()
+    if not target.is_absolute():
+        raise ValueError("'target' must be an absolute path")
+    if not anchor.is_absolute():
+        raise ValueError("'anchor' must be an absolute path")
 
     anchor_copy = Path(str(anchor))
     levels_deep = 0
     while not is_relative_to(anchor_copy, target):
         levels_deep += 1
-        assert anchor_copy != anchor_copy.parent
```

Comment:
Does this ever actually happen?  I could not produce this error so I removed it.

If this actually happens, may I have the scenario so I can handle it and write a test for it?

Reply:
I usually use asserts in scenarios where I'm _pretty sure_ the condition won't happen, but still want to catch it if it does

Reply:
It's just that this `python` command option: https://docs.python.org/3/using/cmdline.html#cmdoption-o
Causes `assert` statements to get removed, which I don't think we want.